src/PortingAssistant.Client.Telemetry/TelemetryCollector.cs

```diff
@@ -205,6 +210,8 @@ string projectGuid
                 string tag
             )
         {
+            if (_disabledMetrics) { return; }
```

Comment:
instead of return on every granular function is there a centralized place you want to add this check?